Source/JavaScriptCore/wasm/WasmOMGIRGenerator.cpp

```diff
@@ -4766,12 +4768,16 @@ B3::PatchpointValue* OMGIRGenerator::createTailCallPatchpoint(BasicBlock* block,
     clobbers.exclude(RegisterSetBuilder::stackRegisters());
     patchpoint->clobber(clobbers);
     patchpoint->clobberEarly(RegisterSetBuilder::macroClobberedGPRs());
-
-    patchpointFunctor(patchpoint, nullptr);
+    patchpoint->appendVector(WTFMove(patchArgs));
     patchpoint->appendVector(WTFMove(constrainedArguments));
 
     block->append(patchpoint);
-    return patchpoint;
+
+    return { patchpoint, nullptr, createSharedTask<B3::StackmapGeneratorFunction>(
+        [wasmCalleeInfoAsCallee, wasmCallerInfoAsCallee, newFPOffsetFromFP] (CCallHelpers& jit, const B3::StackmapGenerationParams& params) {
```

Comment:
Remove space between `[wasmCalleeInfoAsCallee, wasmCallerInfoAsCallee, newFPOffsetFromFP]` and `(...)`.